[0m21:46:33.771207 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108fcea40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10ae83d90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10ae82fb0>]}


============================== 21:46:33.774151 | d6b91edd-54fa-4e20-88a8-da8ad7f2dfb9 ==============================
[0m21:46:33.774151 [info ] [MainThread]: Running with dbt=1.7.19
[0m21:46:33.774541 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': '/Users/chiquang/.dbt', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'invocation_command': 'dbt docs serve --port 8001', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m21:46:33.774927 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/chiquang/Documents/project/Token_metrics/tm-data-pipeline-poc/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m21:46:33.797905 [debug] [MainThread]: Resource report: {"command_name": "serve", "command_wall_clock_time": 0.06606771, "process_user_time": 1.224175, "process_kernel_time": 0.156734, "process_mem_max_rss": "96714752", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m21:46:33.798581 [debug] [MainThread]: Command `dbt docs serve` failed at 21:46:33.798467 after 0.07 seconds
[0m21:46:33.798857 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x108fcea40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10ae83400>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x10ae828f0>]}
[0m21:46:33.799106 [debug] [MainThread]: Flushing usage events
